big_tests/tests/gdpr_SUITE.erl

```diff
@@ -376,14 +406,23 @@ offline_required_modules() ->
     [{mod_offline, [{backend, pick_enabled_backend()}]}].
 
 pubsub_required_modules() ->
+    pubsub_required_modules([<<"flat">>, <<"pep">>, <<"push">>]).
+pubsub_required_modules(Plugins) ->
     [{mod_caps, []}, {mod_pubsub, [
                                    {backend, mongoose_helper:mnesia_or_rdbms_backend()},
                                    {host, "pubsub.@HOST@"},
                                    {nodetree, <<"tree">>},
-                                   {plugins, [<<"flat">>, <<"pep">>, <<"push">>]}
+                                   {plugins, Plugins}
                                   ]
                      }].
 
+mim2_started() ->
```

Comment:
small change but better name would be `is_mim2_started`

Reply:
👍